lib/common/model.js

```diff
@@ -197,9 +195,6 @@ function ModelFactory (
             }
             var update = args.shift();
 
-            // encrypt password fields
-            sanitizer.encrypt(update);
```

Comment:
I'm confused, is this encrypted elsewhere, or are we now just storing decrypted values in the database?